README.md

```diff
@@ -1,5 +1,49 @@
+
+Notebooks are organized according to instrument and observing mode.  Each notebook is designed to process data from uncalibrated raw FITS files to end-stage Level 3 data products (calibrated imaging mosaics, 3-D data cubes, 1-D extracted spectra, etc.).  These notebooks by default run in 'demo' mode, for which they will download and process example data drawn from the [MAST archive](https://archive.stsci.edu/).  They are, however, designed to be simple to run on arbitrary local data sets as well by configuring input directories accordingly.
+
+These notebooks are modular, allowing users to enable or disable different stages of processing.  Likewise, they provide examples of how to customize pipeline processing for specific science cases. 
+
+The following table summarizes the notebooks currently available:
+
+| Instrument | Observing Mode | JWST Build | Minimum ``jwst`` version | Notes                                         |
+|------------|----------------|------------|--------------------------|-----------------------------------------------|
+| MIRI       | MRS            | 11.0       | 1.15.1                   |                                               |
+| NIRISS     | Imaging        | 11.0       | 1.15.1                   |                                               |
+
+While each notebook has been constructed for use with a given minimum version of the ``jwst`` pipeline software, they are generally compatible with more-recent versions (see the JWST pipeline [version history](https://jwst-docs.stsci.edu/jwst-science-calibration-pipeline/jwst-operations-pipeline-build-information)).
+
+## Installation
+
+### Individual Notebooks
+
+These notebooks can be downloaded individually from the GitHub repository and run in any python environment in which the [``jwst``](https://github.com/spacetelescope/jwst) package meeting the indicated minimum version has been installed.  Note that some notebooks have additional dependencies (e.g., [jdaviz](https://github.com/spacetelescope/jdaviz/)) as given in the associated requirements files.
+
+### Package Installation
+
+If desired, you can also clone the entire ``jwst_pipeline_notebooks`` repository to your local computer and set up a new virtual or conda environment
+to avoid version conflicts with other packages you may have installed, for example::
+
+    conda create -n jpnb python=3.9
+    conda activate jpnb
+    git clone https://github.com/spacetelescope/jwst_pipeline_notebooks.git
+
+Next, move into the directory of the notebook you want to install and set up the requirements::
+
+    cd jwst_pipeline_notebooks/notebooks/<whatever-notebook>
+    pip install -r requirements.txt
```

Comment:
The repo is probably more uniquely focused than others, but do we want to go ahead and tell them to create a clean environment using each requirements.txt for the notebook they want to use? So that folks dont create one env, and install the requirements from each notebook repeatedly. It's unlikely in this repo, but that makes it as clean as possible, without dependency conflicts, esp if they start downloading individual notebooks  at different times and not keeping an updated clone.

Reply:
Not sure I followed.  My original intent was to only tell folks to download the notebooks individually and run them in already-set jwst python environments.  The intent behind this package-install option is for anyone looking for an introductory way to run something out of this repo who doesn't necessarily know much about the pipeline.  (sidebar- we probably need to add jupyter to our list of requirements...)

This would be easier if we just had one top level list of requirements for all notebooks, at least on the user side.

Thoughts @rizeladiaz ?  I know you have opinions here.

Reply:
Use case  saw when I read the text:  user may or may not checkout the repo, they download a zip or just the notebook they want.  Sometime later, they do the same, but it's a different version of the notebook and/or jwst cal.  It's cleaner create a new environment for each notebook, and after a long time of not using it. If each notebook has their own requirements file, there can be different software and dependencies installed for each. 

The CI workflows are predicated on the requirements files.  I believe last I looked at the workflows, if we want to enforce all notebooks using the same requirements file, then we have to make sure a common one is used ourselves, and the ones in each notebook directory match. We discussed a global requirements file for the repo, but I don't think the CI workflows currently support that.

Reply:
Ah, I see- you're suggesting that they set up a different environment every time (wasn't sure if you were for or against this).  How's that different from what the text already says though, other than explicitly stating that users should create new environments for every notebook other than implying it?

Reply:
Current notebook repos  store each notebook in a separate directory and each has a requirements file. The CI expects this. The readme suggests users can download notebooks individually and pip install into a single environment the requirments file,  which could contain different things for different notebooks. If they do this repeadetly it could lead to issues. My personal preference is creating the env from the requirements yml that points to the explicit conda and pip packages to install. That's not how the CI is currently setup, but I'd like to avoid users getting into the habit of creating a single environment and installing different requirement files into it with pip. Esp if they, over time, download different notebooks, or different versions of the same notebook.


The catch is users creating environments that they reuse over long-ish periods of time, and downloading notebook updates separately. They need to make sure the environments contain expected software+ versions.

Reply:
How would you suggest rephrasing the 'Package Installation' approach then?

Reply:
I would change the instructions to create a new conda env for each notebook, and install the requirements in it. More advanced folks can choose to ignore this. We should continue to make sure the requirements files are not pinned if individual notebooks support multiple versions.